etl/transform.py

```diff
@@ -89,7 +107,9 @@ def transform(self) -> DataFrame:
             )
         )
 
-        return with_recommendations.union(without_recommendations).withColumn("instance_of", self.instance_of)
+        return with_recommendations.union(without_recommendations)\
+            .withColumn("instance_of", self.instance_of)\
+            .withColumn("filter_out", self.filter_out)
```

Comment:
Just bouncing around some thoughts: is `filter_out` the most descriptive name here? In the generic sense, aren't we asserting that the article is "invalid" (for some definition of valid), and filtering it out is required by the specifics of one use case we are serving?

We might have other use cases (and dataset variants), that are interested in preserving disambiguation pages & c.

Reply:
Hmm, you're right. Would valid be better?

Reply:
Mmm... how about `is_article_page` (following the naming convention of https://www.wikidata.org/wiki/Q15138389)? 

Anyway, it's not a huge deal if we don't get it correct right away. This is an internal API (we don't expose the table as is to client teams - yet).

Reply:
I like that. I've updated it to `is_article_page`